graph/schema.graphqls

```diff
@@ -214,6 +217,7 @@ enum DependencyType {
 
 extend type Query {
   apps: [App!]
+  lengthOfTestCases(app: String): TcCount!
```

Comment:
@Sarthak160 `lengthOfTestCases` is not the right way to send metadata about results. We should add a separate metadata model in the existing queries themselves. We can skip this for now and implement this later to focus on more important items